_data/madewith.json

```diff
@@ -0,0 +1,62 @@
+[
+    {
+        "name": "Browser UIs",
+        "projects":
+        [
+            {
+                "key": "servoshell",
+                "name": "Servoshell",
+                "link": "https://github.com/servo/servo/tree/main/ports/servoshell",
+                "img": "/img/made-with/servoshell.png",
+                "description": "First-party browser UI built with egui and targeting the testing use case"
+            },
+
+            {
+                "key": "verso",
+                "name": "Verso",
+                "link": "https://github.com/versotile-org/verso/",
+                "img": "/img/made-with/verso.png",
+                "description": "A browser UI for Servo that implements the UI as HTML using Servo's own web rendering support"
+            },
+            {
+                "key": "kdab",
+                "name": "Servo Qt",
+                "link": "https://www.kdab.com/embedding-servo-in-qt/",
+                "img": "/img/made-with/servo-qt.png",
+                "description": "A Servo Webview as Qt widget, enabling browser UI built in Qt"
+            },
+            {
+                "key": "moto",
+                "name": " Moto",
+                "link": "https://github.com/moto-browser/moto",
+                "description": "A fork/extension of servoshell that aims to add features such as bookmarks, history and download management."
+            },
+            {
+                "key": "cuervo",
+                "name": "Cuervo",
+                "link": "https://github.com/mcclure/cuervo/tree/unstable_traversal_experiment",
+                "description": "A TUI interface for Servo"
+            }
+        ]
+    },
+    {
+        "name": "Made with Servo Components",
+        "projects":
+        [
+            {
+                "key": "blitz",
+                "name": "Blitz",
+                "link": "https://github.com/DioxusLabs/blitz",
+                 "img": "/img/made-with/blitz.png",
+                 "description": "An HTML renderer and application runtime built on top of Stylo"
+            },
+            {
+                "key": "wry-backend",
+                "name": "Wry Backend",
+                "link": "https://github.com/tauri-apps/wry/tree/servo-wry-demo",
+                 "img": "/img/made-with/wry-backend.png",
+                 "description": "A Servo backend for Wry which abstracts over different webviews"
```

Comment:
Maybe we want to explain what's Wry to associate it with Tauri which a more well-known project (e.g. a *cross-platform webview library for Tauri*).